--- node_modules/panzoom/lib/svgController.js	(date 1581902841611)
+++ node_modules/panzoom/lib/svgController.js	(date 1581902841611)
@@ -61,6 +61,9 @@
   }

   function applyTransform(transform) {
+    transform.scale = Math.round((transform.scale + Number.EPSILON) * 500) / 500;
+    transform.x = Math.round(transform.x);
+    transform.y = Math.round(transform.y);
     svgElement.setAttribute('transform', 'matrix(' +
       transform.scale + ' 0 0 ' +
       transform.scale + ' ' +